srcCpp/RTIDDSImpl.cxx

```diff
@@ -1172,7 +1174,8 @@ class RTIDynamicDataPublisher : public IMessagingWriter
         return (unsigned int)status.pulled_sample_count;
     };
 
-    void wait_for_acknowledgments(const DDS_Duration_t & timeout){
+    void wait_for_acknowledgments(long sec, unsigned long nsec){
+        DDS_Duration_t timeout = {sec, nsec};
```

Comment:
Could you add a space between ")" and "{"

Reply:
Done